### DISCUSSION TITLE
Auto-import suggests deprecated typing

### DISCUSSION THREAD

**Discussion by dvoraen** [microsoft/pylance-release#3722] [Q&A] [answered]
Hi!  I'm asking here because I don't know if this is correct behavior or if it's really an issue or if I'm in the wrong place.

When doing typing for my current project, the auto-import feature is suggesting the deprecated types from `typing` (e.g. `Iterable`, `Callable`) rather than `collections.abc`.  Does Pylance simply find a relevant type and then stop looking for others?  It doesn't even suggest the `collections.abc` types in the auto-import dropdown, as you can see in the attached image.  Is this supposed to happen?

![image](https://user-images.githubusercontent.com/42825596/205905745-18062a98-e0a8-46d9-88f8-0e2812c8cfa7.png)

**Comment by debonte** [comment, marked as answer]
Duplicate of https://github.com/microsoft/pylance-release/discussions/2637?

**Reply by dvoraen** [reply]
So it is.  My apologies.  I did not look hard enough.
